extensions/notebook/src/dialog/pythonPathLookup.ts

```diff
@@ -97,7 +96,7 @@ export class PythonPathLookup {
 				return value;
 			}
 		} catch (err) {
-			this._outputChannel.appendLine(`Problem encountered getting Python path: ${err}`);
+			// Ignoring this error since it's probably from trying to run a non-existent python executable.
```

Comment:
One of these days it would be nice to figure out a common solution for extension logging with level support - this kind of thing would be useful to have for debugging still.